### PR TITLE
Add suggester to take id and set bibliography_id auto

### PR DESCRIPTION
**Pull request by fernandom06**

### CODE REVIEW

File: plantillas/documentoPlantilla.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ---
-bibliography_id: TODO
+<%* const dv = this.app.plugins.plugins["dataview"].api;
+let bibliographies = dv.pages('"bibliografia"');
+let names = bibliographies.name;
+let ids = bibliographies.id;
+const selectId = await tp.system.suggester(names, ids); -%>
+bibliography_id: <%-* tR += selectId; %>
 type: extract
 subtype: reference
```
Comment (alvaromerinog) [inline plantillas/documentoPlantilla.md]: El subtype sería `document`

**Review by alvaromerinog — CHANGES_REQUESTED** [review]
Cambiar `subtype` a `document` `documentoPlantilla.md`

File: plantillas/documentoPlantilla.md (hunk 1 above)
Comment (fernandom06) [reply plantillas/documentoPlantilla.md]: Done

**Comment by fernandom06** [comment]
Cambiado!

**Pull request merged**